themes/survey/fruity_twentythree/css/variations/theme_blueberry-rtl.css

```
@@ -15488,122 +15488,323 @@ body .imageselect-listitem label::after {
-}
-.success-border {
-  border-color: #5076FF;
-}
-
-@media screen and (max-width: 767px) {
-  .choice-5-pt-radio .list-group-horizontal {
-    flex-direction: column;
+  .no-tables-mobile table.ls-answers tr.ls-heading-repeat {
+    display: none;
   }
 }
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)